### PR TITLE
Branch help j unit tests v1.0

### PR DESCRIPTION
**Pull request by nikkiDEEE**

### CODE REVIEW

**Review by heejet — COMMENTED** [review]
LGTM

File: src/test/java/seedu/clialgo/UiTest.java [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class UiTest {
+
+    public static final String WINDOWS_WHITESPACE = "\r\n";
+    public static final String MAC_WHITESPACE = "\n";
```
Comment (ong-ck) [inline src/test/java/seedu/clialgo/UiTest.java:13]: Great way to write constants!

File: src/test/java/seedu/clialgo/UiTest.java [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+
+        if (os.contains("Windows")) {
+            expectedOutput = "======================================================" + WINDOWS_WHITESPACE +
+                    "Hello! Welcome to CLIAlgo Notes!" + WINDOWS_WHITESPACE +
+                    "======================================================" + WINDOWS_WHITESPACE;
+        } else {
+            expectedOutput = "======================================================" + MAC_WHITESPACE +
+                    "Hello! Welcome to CLIAlgo Notes!" + MAC_WHITESPACE +
+                    "======================================================" + MAC_WHITESPACE;
+        }
```
Comment (ong-ck) [inline src/test/java/seedu/clialgo/UiTest.java:34]: Good to have included tests for different os systems!

**Review by ong-ck — COMMENTED** [review]
LGTM! Great job!

**Pull request merged**